crates/task-impls/src/transactions.rs

```diff
@@ -254,29 +282,52 @@ where
         }
         None
     }
+}
 
+impl<
+        TYPES: NodeType,
+        I: NodeImplementation<
+            TYPES,
+            Leaf = SequencingLeaf<TYPES>,
+            ConsensusMessage = SequencingMessage<TYPES, I>,
+        >,
+        A: SequencingConsensusApi<TYPES, SequencingLeaf<TYPES>, I> + 'static,
+    > TransactionTaskState<TYPES, I, A>
+where
+    CommitteeEx<TYPES, I>: ConsensusExchange<
+        TYPES,
+        Message<TYPES, I>,
+        Certificate = DACertificate<TYPES>,
+        Commitment = TYPES::BlockType,
+    >,
+{
```

Comment:
Why a new impl block here? Did you relax trait bounds or something? Please comment.

Reply:
Yes! One of them has additional bounds on `TYPES: NodeType`. I'll add a comment.